### ISSUE TITLE
SSM Overrides $mode-parameter and therefore breaks core functionality

### ISSUE THREAD

**Issue by kanduvisla** [symphonists/subsectionmanager#190]
I came across this little bug when trying to find out why my Bi-Link-field wasn't working.

Turns out that when I have a Bi-Link-field like `talents: entries`, the `__prepareSubsection()`-function is replacing the `: entries`-part with the handle of the section. Therefore the `$mode`-parameter which is later on used by the cores' `appendFormattedElement()`-function is replaced.

I believe this has to do with your TODO-note on line 245 of `extension.driver.php`. Now this causes the Bi-Link field to fail, but it probably causes all other field which do something with `$mode` to fail as well, because SSM makes no difference in the elements described in `$dsParamINCLUDEDELEMENTS`.

I Quick fixed it now by changing line:

```
if($field != 'formatted' && $field != 'unformatted' && $field != 'increment' && !empty($field)) {
```

to:

```
if($field != 'formatted' && $field != 'unformatted' && $field != 'increment' && $field != 'items' && $field != 'entries' && $field != 'count' && !empty($field)) {
```

but, I guess this should get some attention.

**Comment by nilshoerrmann** [comment]
A part of the problem should be fixed in #183.

**Comment by nilshoerrmann** [comment]
For your information: 
- [Discontinuing Support for Subsection Manager](http://getsymphony.com/discuss/thread/83270/7/#position-121)
- [Investigating Relationships](http://getsymphony.com/discuss/thread/96191/)
